ena_portal_api/ena_handler.py

```diff
@@ -55,26 +66,30 @@ def post_request(self, data):
             response = requests.post(self.url, data=data, **get_default_connection_headers())
         return response
 
-    def get_study(self, study_sec_acc):
+    # Supports ENA primary and secondary study accessions
+    def get_study(self, study_acc):
         data = get_default_params()
         data['result'] = 'study'
         data['fields'] = 'study_accession,secondary_study_accession,study_description,study_name,study_title,' \
```

Comment:
We should have an option to requests fields of interest. Default should be everything.